src/components/Dropdowns/ColorRampDropdown.tsx

```diff
@@ -20,10 +20,16 @@ import styles from "./ColorRampDropdown.module.css";
 type ColorRampSelectorProps = {
   selectedRamp: string;
   onChangeRamp: (colorRampKey: string, reversed: boolean) => void;
-  colorRamps?: Map<string, ColorRampData>;
+  /**
+   * The keys of the color ramps to display, in order. If `null` (default), shows all
+   * the color ramps given by `knownColorRamps`.
+   */
+  colorRampsToDisplay?: string[] | null;
+  knownColorRamps?: Map<string, ColorRampData>;
```

Comment:
If the purpose of this feature is to hide these ramps, but make them accessible by URL, why build in this fall back?

Reply:
It was originally in for ease of testing. I've removed it!